src/modules/core/components/SingleUserPicker/SingleUserPicker.css

```diff
@@ -184,4 +184,14 @@
     overflow: hidden;
     text-overflow: ellipsis;
   }
+
+  .container > i {
+    display: flex;
+    padding: 11px 0;
+    border-bottom: 1px solid var(--temp-grey-blue-7);
+  }
+
+  .arrowIconActive {
+    border-bottom-color: var(--primary) !important;
+  }
```

Comment:
This feels hacky to me, could we not do something like this?
![Screenshot from 2022-07-20 15-50-00](https://user-images.githubusercontent.com/18473896/180060344-248ddae2-4c65-48a7-a4ae-0d3141b82b67.png)

This way we don't need the borders (and more importantly, that `!important`) and the static positioning

Reply:
If you switch your dev tools to responsive mode, and change the width of the viewport, you'll notice that absolute positioning does not respond well (at all) to changes in the viewport width. This means that `left: 299px` works only on one viewport width, whereas currently the positioning responds to all screen sizes up to the breakpoint.

Reply:
However, I agree about the `important!`. There are often times in which it's unnecessary, and this is one of them. The specificity issue had a simple fix, now implemented, of using the correct class name to select the arrow.

Reply:
Oh, I see what you are saying. This may indicate that perhaps something else in the styles of the core component needs to change because I feel like this should be more straightforward than how it is right now. 

Incidentally, I'm currently reworking this component so I may have the chance to update the styles so that this may be simpler :)

For now, I feel like this is ok :+1: We could revisit this once the component is updated